packages/api/src/server/role/role.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString } from 'class-validator';
+
+export class RoleDto {
```

Comment:
Please read my comment on AuthActionDto; I also have the same question (point) for this.

Reply:
Yes Sure

Reply:
I've added the [reason](https://github.com/spaship/spaship/pull/2104#discussion_r1070877357) here, please check.